core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/puppeteer-utils.js

```diff
@@ -42,11 +42,11 @@ module.exports = class baseUser {
         this.browserObject = browser;
         this.page = await browser.newPage();
         await this.page.setViewport({ width: 0, height: 0 });
-        // We are doing this generically because the on-alert behaviour needs
-        // to be defined through an event listener, and Puppeteer does not
-        // support handling these alerts on a case-by-case basis.
-        // See (https://github.com/puppeteer/puppeteer/blob/v0.12.0/docs/api.md#class-dialog)
-        // for more information.
+        /** We accept all browser alerts due to empty messages we get from
```

Comment:
No, we don't accept all browser alerts. We accept only browser alerts that have specific messages. One of those error messages is the empty string.

I would like to see a video proof that if we add window.alert('abc') to the main code then that fails the Puppeteer tests.

Reply:
@seanlip please check, the test is failing with the alert text 'abc' as this alert text is not added in the acceptedBrowserAlert texts.

https://user-images.githubusercontent.com/91013793/223348106-27551271-5c09-4f52-8827-3911d612521f.mov